docs/using-the-jdbc-driver/RdsProxy.md

```diff
@@ -0,0 +1,42 @@
+|--------------------------------|-----------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Authentication                 | Supports native authentication, IAM authentication, and AWS Secrets Manager                   | Supports native authentication and IAM authentication to the proxies. The proxies then uses AWS Secrets Manager to access the instances.                                                                                                                                                                                                               |
+| Connection Pooling             | Requires third-party connection pools, such as HikariCP                                       | Supports third-party connection pooling but is also capable of handling the infrastructure to perform connection pooling for the writer instance of the associated RDS or Aurora database.                                                                                                                                                             |
+| Failover Support               | Supports Aurora MySQL and PostgreSQL database engines                                         | Supports all Aurora database engines as well as the RDS database engines                                                                                                                                                                                                                                                                               |
+| Transaction Boundary Detection | The AWS JDBC Driver uses keywords such as COMMIT or ROLLBACK in the SQL statements            | RDS Proxy detects when a transaction ends through the network protocol used by the database client application. For more information, see RDS Proxy's [Transactions page](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy.howitworks.html#rds-proxy-transactions).                                                                    |
+| Security                       | The AWS JDBC Driver follows the same security mechanisms supported by the underlying drivers. | RDS Proxy supports TLS protocol version 1.0, 1.1, and 1.2 for sessions between your client and the RDS Proxy endpoint. To use SSL connections from the RDS Proxy to the database, [you need to set the SSL session variables on the client side](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy.howitworks.html#rds-proxy-security). |
+
+## Additional FAQ
+
+### Can the AWS RDS Proxy be used with Host Monitoring Connection Plugin
```

Comment:
Is it worthwhile to expand on the why? e.g., We make this recommendation, but why do we do so?

Reply:
The why is explained in the linked documentation, not sure if it makes sense to copy the content over here.